src/contants/registrationPages.js

```diff
@@ -7,32 +7,38 @@ const pages = {
   'add-nickname': {
     current: 'add-nickname',
     next: 'add-show-age',
+    previous: 'info',
     skippable: false,
   },
   'add-show-age': {
     current: 'add-show-age',
     next: 'add-location',
+    previous: 'add-nickname',
     skippable: false,
   },
   'add-location': {
     current: 'add-location',
     next: 'add-description',
-    skippable: true,
+    previous: 'add-show-age',
+    skippable: false,
```

Comment:
Skippable is probably no longer necessary here? It was used earlier for "ohita" button, but now as far as I can remember it's no longer used for anything.

Reply:
Got rid off it, thanks :)